index.js

```diff
@@ -327,6 +328,23 @@ class MillerColumnsElement extends HTMLElement {
     const div = document.createElement('div')
 
     if (!root) {
+      // Append back link
+      const backLink = document.createElement('button')
+      backLink.className = this.classNames.backLink
+      backLink.type = 'button'
+      backLink.innerHTML = 'Back'
```

Comment:
As far as I know, this element is i18n friendly until we add this `Back` button - now we have hardcoded English in the JS. Can it be read from a parameter in the `<miller-columns-selected` invocation, perhaps defaulting to `Back` if not provided?

Reply:
i18n was not on the list of things considered at the initial build. This is not the only case where we hardcode content we do it for the "Remove topic" button and "No selected topics" message. So I wouldn't extract it as part of this work, but I agree it's a potential improvement that can be done across the component.

Reply:
👍 cool, didn't realise there was already hardcoded English in the codebase.